README.md

```diff
@@ -29,32 +46,32 @@ Specify a name of the file `File::Find` is ought to look for. If you pass a stri
 type
 ----
 
-Given a type, `find()` will only return files being the given type. The available types are `file`, `dir` or `symlink`.
+Given a type, `find()` will only return files being the given type. The available types are `file`, `dir`, or `symlink`.
 
 exclude
 -------
 
-Exclude is meant to be used for skipping certain big and uninteresting directories, like '.git'. Neither them nor any of their contents will be returned, saving a significant amount of time.
+Specify a regex (or any other smartmatchable type) to exclude files / directories from the search.
 
-The value of `exclude` will be smartmatched against each IO object found by File::Find. It's recommended that it's passed as an IO object (or a Junction of those) so we avoid silly things like slashes vs backslashes on different platforms.
+recursive
+---------
+
+By default, `find` will recursively traverse a directory tree, descending into any subdirectories it finds. This behavior can be changed by setting `recursive` to a false value. In this case, only the first-level entries will be processed.
 
 keep-going
 ----------
 
 Parameter `keep-going` tells `find()` to not stop finding files on errors such as 'Access is denied', but rather ignore the errors and keep going.
 
-recursive
----------
-
-By default, `find` will recursively traverse a directory tree, descending into any subdirectories it finds. This behaviour can be changed by setting `recursive` to a false value. In this case, only the first level entries will be processed.
+**Note: This parameter is currently broken and does not affect the search in any way.**
```

Comment:
It does affect the search. If I remember correctly, we have cleared this up. The test cases might be broken but it basically does the right thing nevertheless.

Reply:
This note has been removed.

Reply:
I still see it - didn't you accidentally remove something else?

Reply:
This is getting hard to read. I'm going to close this PR and rebubmit